projects/Mallard/src/components/front/index.tsx

```diff
@@ -82,12 +88,15 @@ const Page = ({
     const { width } = Dimensions.get('window')
     const translateX = getTranslateForPage(scrollX, index)
     const collectionResponse = useCollectionResponse(issue, collection)
-
+    const [{ hasLiveDevMenu }] = useSettings()
```

Comment:
this refers to toggling the secret dev options in the production app. rename to `isUsingProdDevtools` maybe?